CONTRIBUTING.md

```diff
@@ -37,3 +37,30 @@ For example, for running tests in `multi`, `uuid` and `main` packages, call
 ```bash
 make test-multi test-uuid test-main
 ```
+
+## Code review checklist
+
+- Public API contains functions, variables, constants that are needed from
+  outside by users. All the rest should be left closed.
+- Public functions, variables and constants contains at least a single-line
+  comment.
+- Code is DRY (see "Do not Repeat Yourself" principle).
+- New features have functional and probably performance tests.
+- There are no changes in files not related to the issue.
+- There are no obvious flaky tests.
+- Commits with bugfixes have tests based on reproducers.
+- Changelog entry is present in `CHANGELOG.md`.
+- Public methods contains executable examples (contains a comment with
+  reference output).
+- Autogenerated documentation looks good. Run `godoc -http=:6060` and point
+  your web browser to address "http://127.0.0.1:6060" for evaluating.
+- Commit message header starts with prefix and short description follows after
+  colon.
```

Comment:
Prefixing commit titles is the nice thing for a project with many components, because it allows to easily spot what is changed in a particular commit. In a small project like this connector many commits are actually about 'this project' component, so I would not make this rule mandatory. We can suggest to mark readme/documentation/test/ci commits explicitly and leave others prefixed if it makes sense. It would be good to write examples of prefixes right here.

Reply:
Looks reasonable.

From the other side, commit headers *with* and *without* prefixes will look inconsistent in a Git history.

Updated:

```diff
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -54,8 +54,10 @@ make test-multi test-uuid test-main
   reference output).
 - Autogenerated documentation looks good. Run `godoc -http=:6060` and point
   your web browser to address "http://127.0.0.1:6060" for evaluating.
-- Commit message header starts with a prefix and a short description follows after
-  colon.
+- Commit message header may start with a prefix with a short description
+  follows after colon. It is applicable to changes in a README, examples, tests
+  and CI configuration files. Examples: `github-ci: add Tarantool 2.x-latest`
+  and `readme: describe how to run tests`.
 - Check your comments, commit title, and even variable names to be
   grammatically correct. Start sentences from a capital letter, end with a dot.
   Everywhere - in the code, in the tests, in the commit message.
```